src/Evolution/Systems/ForceFree/MaskNeutronStarInterior.hpp

```diff
@@ -0,0 +1,89 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <type_traits>
+
+#include "DataStructures/DataBox/Protocols/Mutator.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/EagerMath/DotProduct.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/Tags.hpp"
+#include "Evolution/DgSubcell/Tags/Coordinates.hpp"
+#include "Evolution/DgSubcell/Tags/Inactive.hpp"
+#include "Evolution/Systems/ForceFree/SimulationType.hpp"
+#include "Evolution/Systems/ForceFree/Tags.hpp"
+#include "PointwiseFunctions/AnalyticData/ForceFree/Factory.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/ForceFree/Factory.hpp"
+#include "PointwiseFunctions/InitialDataUtilities/InitialData.hpp"
+#include "PointwiseFunctions/InitialDataUtilities/Tags/InitialData.hpp"
+#include "Utilities/CallWithDynamicType.hpp"
+#include "Utilities/ConstantExpressions.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/ProtocolHelpers.hpp"
+#include "Utilities/SetNumberOfGridPoints.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace ForceFree {
+/*!
+ * \brief Assign the masking scalar variable (see Tags::NsInteriorMask) at the
+ * initialization phase in NS magnetosphere simulations.
+ *
+ * For an isolated NS magnetosphere simulations, center of the NS is located at
+ * the origin and coordinate radius of the NS is rescaled to 1. Grid points
+ * with $x^2 + y^2 + z^2 < 1.0$ are identified as interior, and are subject to
+ * correction of evolved variables in each time steps to enforce the MHD
+ * condition (see ImposeMhdConditionInsideNs)
+ *
+ * \note The location of NS interior (mask) is fixed in the Grid frame to avoid
+ * the NS surface crossing grid points during the evolution.
```

Comment:
Use the inertial frame because that's how the fields are defined.  The result can't vary from call to call because initialization mutators are only called once.

Reply:
Hmm maybe I'll remove this `\note` comment and use inertial frame coordinates for initialization.